packages/scxa-experiment-page/package.json

```diff
@@ -53,12 +54,16 @@
     "babel-loader": "^8.1.0",
     "clean-webpack-plugin": "^3.0.0",
     "eslint": "^8.17.0",
+    "eslint-plugin-cypress": "^2.15.2",
     "eslint-plugin-jest": "^28.8.3",
     "file-loader": "^6.1.0",
-    "image-webpack-loader": "^7.0.0",
-    "webpack": "^4.44.1",
-    "webpack-cli": "^3.3.12",
-    "webpack-dev-server": "^3.11.0",
+    "image-webpack-loader": "^4.5.0",
```

Comment:
Why are we downgrading this dependency?

Reply:
I think that's an automatic fix from npm audit fix. I will bring it back to version 7.